src/einsteinpy/constant.py

```diff
@@ -1,6 +1,10 @@
 import numpy as np
 from astropy import constants, units as u
 
+__all__ = ["c", "G", "eps0", "coulombs_const",
+           "Cosmo_Const_base", "Cosmo_Const",
```

Comment:
`Cosmo_Const_base` and `Cosmo_Const` removed

Reply:
I apologize, I was unclear. I meant to say, you can remove `Cosmo_Const_base` from `__all__`, because only `Cosmo_Const` needs to be exposed, since it is an instance of `astropy.units.Quantity` like `c`, `G` and the rest of the constants.  `Cosmo_Const_base` is an intermediary `float`, which has been used to define `Cosmo_Const`. You can add `Cosmo_Const` back to `__all__`.